crates/bevy_ecs/src/world/mod.rs

```diff
@@ -2085,6 +2085,19 @@ impl World {
         self.storages.resources.clear();
         self.storages.non_send_resources.clear();
     }
+
+    /// Initializes all of the components in the given [`Bundle`] and returns their ids.
+    ///
+    /// This is equivalent to calling [`init_component`](Self::init_component) on each component
+    /// in the bundle.
+    #[inline]
+    pub fn init_bundle<B: Bundle>(&mut self) -> Vec<ComponentId> {
+        let mut ids = Vec::new();
```

Comment:
This doesn't register the bundle in `Bundles`, which can probably save some effort in constructing the Vec by just cloning the one in BundleInfo instead of checking components and storage for it.

Reply:
That makes sense, I've switched it over to using `Bundles.init_info`, although I guess it could maybe also make sense to return a `BundleInfo` in case someone wants the id of the bundle as well?

Reply:
Yep, returning a `&BundleInfo` is probably the right thing to do here to provide more complete access to the metadata.